### ISSUE TITLE
Issue with "background-position": 2-value syntax behaves differently than browser

### ISSUE THREAD

**Issue by kalokng** [mdn/content#12008]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/background-position

#### What information was incorrect, unhelpful, or incomplete?
the meaning of parameters in 2-value syntax behaves in browsers is different than the doc.

#### Specific section or headline?
https://developer.mozilla.org/en-US/docs/Web/CSS/background-position#values

#### What did you expect to see?
according to the doc,
```
background-position: bottom 20%;
```
should means the background starts at the bottom, and 20% from left.

#### Did you test this? If so, how?
`bottom 20%` is `invalid property value` in Chrome, Firefox, Edge and IE11.
`50px right` is also `invalid property value`

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/css/background-position`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/background-position
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/background-position/index.md
* Last commit: https://github.com/mdn/content/commit/9ca237eb91173fa84eb24eb5a6336efc14f18f62
* Document last modified: 2021-08-12T23:59:09.000Z

</details>

**Comment by ibrahim-301** [comment]
According to the [W3C Candidate Recommendation Draft, 26 July 2021](https://drafts.csswg.org/css-backgrounds/#propdef-background-position)

> Note that a pair of keywords can be reordered while a combination of keyword and length or percentage
> cannot. So `center left` is valid while `50% left` is not. 

So yes the docs should probably clarify that for the **2-value syntax**, if both values are keywords then the order doesn't matter.
Both of these are valid:

```css
background-position: right top;
```

```css
background-position: top right;
```

On the other hand, if one value is a length/percentage and the other one is a keyword then the first value should define the X while the second value defines the Y:

```css
background-position: right 20%; /* First value can be [ right | center | left ] */
```
```css
background-position: 20% top; /* Second value can be [ top | center | bottom ] */
```

**Comment by Anurella** [comment]
@kalokng The **2-value syntax** considers order when pairing a keyword with a percentage/length value @ibrahim-301 gives a great example.  You can use all keywords or all numbers to prevent confusion or use the **3-value syntax** if you need to offsets.

**Comment by Anurella** [comment]
@sideshowbarker Will adding the extra information be helpful in closing this issue? If yes you can assign it to me.

**Comment by sideshowbarker** [comment]
@Anurella 👍 OK, assigned

**Comment by NiedziolkaMichal** [comment]
This issue can be closed, since #12854  fixed it.

**Comment by sideshowbarker** [comment]
Closing per https://github.com/mdn/content/issues/12008#issuecomment-1060030392
